parquet_fe_prototype/search.py

```diff
@@ -65,10 +88,17 @@ def run_search(ix: index, raw_query: str) -> list[Resource]:
         parser = MultifieldParser(
             ["name", "description", "columns"],
             ix.schema,
-            fieldboosts={"name": 2.0, "description": 1.0, "columns": 0.5},
+            fieldboosts={"name": 1.5, "description": 1.0, "columns": 0.5},
         )
         query = parser.parse(raw_query)
-        out_boost = Term("tag", "out", boost=2.0)
-        preliminary_penalty = Term("tag", "preliminary", boost=-5.0)
-        results = searcher.search(Or([query, out_boost, preliminary_penalty]))
+        out_boost = Term("tags", "out", boost=10.0)
```

Comment:
We had to actually search for the `"tags"` field not the `"tag"` field. Plus the boost values were *way* too small to do anything in the actual search results.